code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -306,7 +303,6 @@
 	name = "medical hardsuit"
 	desc = "A special suit that protects against hazardous, low pressure environments. Built with lightweight materials for easier movement."
 	item_state = "medical_hardsuit"
-	slowdown = 1
```

Comment:
Medical hardsuit has no special quality with this change. Maybe you want to set it to 0 slowdown in this case? I do not know if it takes floats instead of integers.